services/community-site/src/services/useExplorerStats.tsx

```diff
@@ -32,9 +34,18 @@ export default () => {
           const lastBlockDate = new Date(lastBlockTimestamp * 1000);
           const diff = new Date().getTime() - lastBlockDate.getTime();
           const diffHours = diff / 1000 / 60 / 60;
+          const producedBlocksInLast24hours = diffHours < 24;
+          const isEligible = await isValidatorEligible(validator.address);
+          const validatorStatus = isEligible
+            ? producedBlocksInLast24hours
+              ? ValidatorStatus.Green
+              : ValidatorStatus.Yellow
+            : ValidatorStatus.Grey;
+
           return {
             ...validator,
-            isActive: diffHours < 24,
+            isActive: producedBlocksInLast24hours,
```

Comment:
Do we need this anymore?

Reply:
Yes, having this isActive boolean might be usefull. I noticed it used in 1 or 2 places so that`s why I left it.